src/Tests/Microsoft.NET.Sdk.Web.Tests/PublishTests.cs

```diff
@@ -55,16 +60,26 @@ public void TrimmingOptions_Are_Defaulted_Correctly_On_Aot_Apps(string targetFra
 
             var testProject = CreateTestProjectForILLinkTesting(targetFramework, projectName);
             testProject.AdditionalProperties["PublishAOT"] = "true";
+            testProject.AdditionalProperties["RuntimeIdentifier"] = rid;
+            testProject.PropertiesToRecord.Add("PublishTrimmed");
+            testProject.PropertiesToRecord.Add("TrimMode");
+            testProject.PropertiesToRecord.Add("PublishIISAssets");
 
             var testAsset = _testAssetsManager.CreateTestProject(testProject, identifier: projectName + targetFramework);
             var publishCommand = new PublishCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
-            publishCommand.Execute($"/p:RuntimeIdentifier={rid}").Should().Pass();
+            publishCommand.Execute().Should().Pass();
+
+            var buildProperties = testProject.GetPropertyValues(testAsset.TestRoot, targetFramework);
+            buildProperties["PublishTrimmed"].Should().Be("true");
+            buildProperties["TrimMode"].Should().Be("");
```

Comment:
This verifies we don't set TrimMode to "partial" like we do in the PublishTrimmed-only case. Does anyone know of a better way to verify we get "full" trimming?